doc/API.md

```diff
@@ -261,60 +287,66 @@ provided hash.
 Gateway
 -------
 
-Queries:
-
-* /gateway                     [GET]
-* /gateway/add/{netaddress}    [POST]
-* /gateway/remove/{netaddress} [POST]
-
-#### /gateway
-
-Function: Returns information about the gateway, including the list of peers.
-
-Parameters: none
-
-Response:
-```
-struct {
-	netaddress string
-	peers      []struct {
-                netaddress string
-                version    string
-                inbound    bool
-        }
+| Route | HTTP verb |
+| ----- | --------- |
+| [/gateway](#gateway-get) | GET |
+| [/gateway/connect/{netaddress}](#gatewayconnectnetaddress-post) | POST |
+| [/gateway/disconnect/{netaddress}](#gatewaydisconnectnetaddress-post) | POST |
```

Comment:
Ideally this would be cleaner in markdown, can you align the bars? I believe it will still format correctly.